invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/uploads_items/ComputerTabletUploadsItem.js

```diff
@@ -88,6 +80,16 @@ export const ComputerTabletUploadsItem = ({
               icon="eye"
               content={i18next.t("View")}
             />
+          ) : (
+            <Button
+              compact
+              size="small"
+              floated="right"
+              onClick={() => viewDraft()}
```

Comment:
Question: Does this mean that from the list view for the records  we cannot access the edit anymore?

Reply:
Yes that is one of the big changes also mentioned in https://github.com/inveniosoftware/invenio-rdm-records/issues/1948